src/main/kotlin/lotto/rank/LottoRank.kt

```diff
@@ -1,17 +1,53 @@
 package lotto.rank
 
+import lotto.Lotto
+import lotto.statistics.WinningNumber
+
 enum class LottoRank(
     val matchCount: Int,
     val prize: Int,
+    val matchBonus: Boolean = false,
 ) {
     FIRST(6, 2_000_000_000),
-    SECOND(5, 1_500_000),
-    THIRD(4, 50_000),
-    FOURTH(3, 5_000),
+    SECOND(5, 30_000_000, true),
```

Comment:
```suggestion
    SECOND(5, 30_000_000, matchBonus = true),
```
다른 개발자들도 어떤 의미인지 한 눈에 이해할 수 있도록 Kotlin의 [named arguments](https://kotlinlang.org/docs/functions.html#named-arguments)를 활용해보시면 어떨까요~?

Reply:
다른 개발자가 봤을때 훨씬 쉽게 파악할 수 있을 것 같아요 👍  다음 미션에서 수정하도록 하겠습니다!